modules/clair-unmanaged.adoc

```diff
@@ -38,24 +38,17 @@ The {productname} Operator for OpenShift Container Platform allows users to prov
 
 .Procedure
 
-. Create a `clair-config.yaml` bundle secret:
+. Create a Quay config bundle secret that includes the `clair-config.yaml`:
 +
 [source,terminal]
 ----
-$ oc create secret generic --from-file config.yaml=./config.yaml --from-file extra_ca_cert_rds-ca-2019-root.pem=./rds-ca-2019-root.pem --from-file clair-config.yaml=./clair-config-aws-rds-postgres_ca_cert.yaml --from-file ssl.cert=./ssl.cert --from-file ssl.key=./ssl.key config-bundle-secret
+$ oc create secret generic --from-file config.yaml=./config.yaml --from-file extra_ca_cert_rds-ca-2019-root.pem=./rds-ca-2019-root.pem --from-file clair-config.yaml=./clair-config.yaml --from-file ssl.cert=./ssl.cert --from-file ssl.key=./ssl.key config-bundle-secret
 ----
 +
 Example `clair-config.yaml` configuration:
 +
 [source,yaml]
 ----
-auth:
-    psk:
-       iss:
-          - quay
-          - clairctl
-       key: <example_key>
-http_listen_addr: :8080
 indexer:
     connstring: host=quay-server.example.com port=5432 dbname=quay user=clairuser password=clairpass sslrootcert=/run/certs/rds-ca-2019-root.pem sslmode=verify-ca
```

Comment:
if we can reminder customer that this point "sslrootcert=/run/certs/", the cert path must under "/run/certs/", it would be better